--- /etc/lightdm/lightdm-gtk-greeter.conf	2022-10-28 08:55:42.856823295 -0500
+++ /etc/lightdm/lightdm-gtk-greeter.conf	2022-10-28 11:58:51.969957144 -0500
@@ -48,17 +48,17 @@
 #  transition-duration = overrides default value
 #
 [greeter]
-#background=
-#user-background=
-#theme-name=
-#icon-theme-name=
-#font-name=
+background = #53585B
+user-background = true
+theme-name = nGUI-Pro
+icon-theme-name = nGUI-Pro
+font-name = Bitstream Vera Sans Roman 10
 #xft-antialias=
 #xft-dpi=
 #xft-hintstyle=
 #xft-rgba=
-#indicators=
-#clock-format=
+indicators = ~session;~spacer;~host;~spacer;~clock;~power
+clock-format =   %a %b %d,  %H:%M  
 #keyboard=
 #reader=
 #position=
