--- /etc/lightdm/lightdm-gtk-greeter.conf	2021-01-10 04:52:31.825055906 -0600
+++ /etc/lightdm/lightdm-gtk-greeter.conf	2022-02-25 10:07:15.000000000 -0600
@@ -48,16 +48,16 @@
 #  transition-duration = overrides default value
 #
 [greeter]
-#background=
-#user-background=
-#theme-name=
-#icon-theme-name=
-#font-name=
+background = #53585B
+user-background = true
+theme-name = nGUI-DarkBlue
+icon-theme-name = nGUI-Blue
+font-name = Bitstream Vera Sans Roman 10
 #xft-antialias=
 #xft-dpi=
 #xft-hintstyle=
 #xft-rgba=
-#indicators=
+indicators = ~host;~spacer;~clock;~spacer;~session;~power
 #clock-format=
 #keyboard=
 #reader=
